### DISCUSSION TITLE
This forum is dead... active Z2M forum somewhere else?

### DISCUSSION THREAD

**Discussion by Ltek** [Koenkk/zigbee2mqtt#18916] [General]
I started using Z2M a month ago and posted a few times here... crickets. No Help at all. Is this software just not that widely used or is there a Z2M forum somewhere else that is active?  I like the app so far but dont want to hitch my wagon to a place where there is a tiny community and/or no help when we have questions

**Comment by hyperion007** [comment]
I have been using Z2M for a few years and have also noticed the lack of community and activity on this forum. Most people seem to use Home Assistant but I really like the Z2M and node-red setup. Let me know if you find a more alive forum somewhere.

**Comment by MacDefender** [comment]
I started with Z2M about 2 month ago and already love it. Currently automated everything in node-red but might try some other option like ioBroker too. Would love to have a good community/forum too as Z2M is just awesome (running on a NanoPi R2S with Armbian)

**Comment by DubhAd** [comment]
There is also a Discord, though it's probably _less_ active than the discussions here.

**Comment by LemmeDasker** [comment]
I also wonder if the support here only are for commercial products and not DIY. I have a custom device I programmed myself that are sooooo close to work, but unfortunately no support to get from here. I have to abandon the project.

Perhaps a donation would work?

**Reply by hyperion007** [reply]
I fully understand that the contributors don't have time to support in the sense a paid support does. 
I'm mean the overall activity on the forum from users. 
Maybe everyone's Z2M systems work too well so people don't need to hang around here that much :)

**Comment by francisp2** [comment]
I check in almost daily, but most users only come if they have a problem.

**Comment by Ltek** [comment]
The fact that this post has more action than any other many months says something, not in a good way.
I've posted asking simple questions...no replies, not help. I've seen others do so also, going many months back.
A good community is when developers AND users helping each other... daily 
There is 1000x more activity in ZHA and Deconz
I put in a ticket for a feature request rhat is absolutely needed (im surprised its not already available ... its basic but massively useful for users and would cut developerment time/supportfor adding new device definitions) ... I got an reply then silence.

**Comment by lux73** [comment]
the Forums isn't dead - but People really should search before Post!!

Even the more skilled User don't wan't answer the same Questions again and again...

So many Questions are already answered - so please search and read carefully around the Forums before Post any Questions who has already answered 🤐

**Reply by Ltek** [reply]
Welcome to the Internet. Get used to it, that is the way things are. But, that's not what I was talking about. 

This forum should have an exchange of experiences on device use, converters, how Z2M works on different HA systems, etc. Like I did when I found a motor that didnt work 100% using Z2M but did when a converter was added. 

I don't know of an alternative forum with these for Z2M (facebook, etc) so this is the logical place... there is even a specific menu item named "Discussions" ;-)

**Reply by lux73** [reply]
"how Z2M works on different HA systems" -> z2m is not HA & vice versa

me personally is using OpenHAB but i would not ask any Questions regarding any working OH Rule here BUT many HA User seem to ask HA specific Questions in this z2m Forum. I dont know why...

So deal with it 🤫

I could only second: use the Search or your Question will likely not answered
